libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/DefaultAudioTrackBufferSizeProvider.java

```diff
@@ -232,7 +258,13 @@ protected int getPassthroughBufferSizeInBytes(@C.Encoding int encoding, int bitr
     int bufferSizeUs = passthroughBufferDurationUs;
     if (encoding == C.ENCODING_AC3) {
       bufferSizeUs *= ac3BufferMultiplicationFactor;
+    } else if (encoding == C.ENCODING_DTS_HD) {
+      // DTSHD (DTS Express) for streaming uses a frame size (number of audio samples per channel
+      // per frame of 4096. This requires a higher multiple for the buffersize computation.
```

Comment:
nit: A missing closing `)` after the second word "frame"? The corresponding open `(` is in the last line.

Reply:
Done